connexion/decorators/metrics.py

```diff
@@ -30,16 +32,19 @@ def __call__(self, function):
 
         @functools.wraps(function)
         def wrapper(request):
-            status_code = 500
+            status = 'unknown-exception'
```

Comment:
FWIW I can imagine this being a big pain for someone trying parse logs who expects the status to always be an integer.

Reply:
I think I'd still default this to 500, what do you think?

Reply:
I also expect it to be 500, since that's what makes sense from an end-2-end visibility point of view.